build/artifacts.props

```diff
@@ -73,96 +67,111 @@
-    <PackageArtifact Include="Microsoft.AspNetCore.Server.HttpSys" Category="noship" />
-    <PackageArtifact Include="Microsoft.AspNetCore.Server.IIS" Category="noship" />
-    <PackageArtifact Include="Microsoft.AspNetCore.Server.IISIntegration" Category="noship" />
-    <PackageArtifact Include="Microsoft.AspNetCore.Server.IntegrationTesting.IIS" Category="noship" />
-    <PackageArtifact Include="Microsoft.AspNetCore.Server.IntegrationTesting" Category="noship" />
-    <PackageArtifact Include="Microsoft.AspNetCore.Server.Kestrel.Core" Category="noship" />
-    <PackageArtifact Include="Microsoft.AspNetCore.Server.Kestrel.Transport.Abstractions" Category="noship" />
-    <PackageArtifact Include="Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv" Category="noship" />
-    <PackageArtifact Include="Microsoft.AspNetCore.Server.Kestrel.Transport.Sockets" Category="noship" />
-    <PackageArtifact Include="Microsoft.AspNetCore.Server.Kestrel" Category="noship" />
-    <PackageArtifact Include="Microsoft.AspNetCore.Session" Category="noship" />
-    <PackageArtifact Include="Microsoft.AspNetCore.SignalR.Core" Category="noship" />
-    <PackageArtifact Include="Microsoft.AspNetCore.SignalR" Category="noship" />
-    <PackageArtifact Include="Microsoft.AspNetCore.StaticFiles" Category="noship" />
-    <PackageArtifact Include="Microsoft.AspNetCore.WebSockets" Category="noship" />
-    <PackageArtifact Include="Microsoft.AspNetCore.WebUtilities" Category="noship" />
-    <PackageArtifact Include="Microsoft.AspNetCore" Category="noship" />
-    <PackageArtifact Include="Microsoft.Extensions.ApplicationModelDetection" Category="noship" />
-    <PackageArtifact Include="Microsoft.Net.Http.Headers" Category="noship" />
-    <PackageArtifact Include="Microsoft.Web.Xdt.Extensions" Category="shipoob" />
-    <PackageArtifact Include="Internal.WebHostBuilderFactory.Sources" Category="noship"/>
+      <!-- These packages contain CLI tools which are bundled in the .NET Core SDK. -->
+      <PackageArtifact Include="dotnet-dev-certs" Category="noship" />
+      <PackageArtifact Include="dotnet-sql-cache" Category="noship" />
+      <PackageArtifact Include="dotnet-user-secrets" Category="noship" />
+      <PackageArtifact Include="dotnet-watch" Category="noship" />
+
+    <!-- This package contains API for the .NET CLI to generate the aspnet HTTPs dev cert during CLI first-run initialization. -->
+      <PackageArtifact Include="Microsoft.AspNetCore.DeveloperCertificates.XPlat" Category="noship" />
+
+    <!-- This packages contain bits used by Azure site extensions, and are not currently deployed to NuGet.org automatically like the rest of our packages. -->
+      <PackageArtifact Include="AspNetCoreRuntime.3.0.$(SharedFxArchitecture)" Category="noship" Condition=" '$(SharedFxRid)' == 'win-x64' OR '$(SharedFxRid)' == 'win-x86' " />
+      <PackageArtifact Include="Microsoft.Web.Xdt.Extensions" Category="noship" />
```

Comment:
is 'shipoob' still a category?

Reply:
Effectively, no. There hasn't been any real difference between noship and shipoob since we condensed to a single myget feed earlier this year. As a part of arcade convergence, there will only be 2 categories, so I'm moving the last of the shipoob's to noship.

Reply:
They noship items are still signed?

Reply:
So there would be no packages that are _not_ shipped to NuGet, but released in VS?

Reply:
@Tratcher: yes, all packages are signed, regardless of shipping category. 

@muratg: I don't think we ship any NuGet packages in VS only. All packages we ship end up on nuget.org. Most of the 'noship' packages are used to transfer binaries to other internal teams who typically take the binaries out of the package and re-distribute the binary in something else, such as WTE or the .NET Core SDK. I don't know if Microsoft.Web.Xdt.Extension happens to be one of those.

Reply:
Microsoft.Web.Xdt.Extension ships in the site extension.

Reply:
So, is there any need for this to produced as a .nupkg? cc @pakrym

Reply:
not a shipping one, no.

Reply:
We don't need it packaged at all.

Reply:
Ok. I'll follow-up in a separate PR on this one.

Reply:
Wasn't the AI team also consuming it from the package?

Reply:
They seem to have their own set of xdt extensions: https://mseng.visualstudio.com/AppInsights/_git/StatusMonitor?path=%2FExtensions%2FExtension%2FApplicationInsights.XdtExtensions&version=GBmaster

Reply:
@glennc did they stop using ours? Or do they have both?

Reply:
In addition Microsoft.Web.Xdt.Extension has never changed since it's creation so they won't be broken by us stopping producing new versions.